src/test/ui/lint/command-line-lint-tool.rs

```diff
@@ -0,0 +1,4 @@
+// compile-flags: -A clippy::foo
+// build-pass
```

Comment:
```suggestion
// check-pass
```
(No need to test codegen and linking here.)